### PR TITLE
Updating chaos mesh version

### LINKED ISSUE #2662: [stress testing][s] Stress infra helm deployments are failing
When running the [provision script](https://github.com/Azure/azure-sdk-tools/blob/main/tools/stress-cluster/cluster/provision.ps1) there appears to be an issue with the template and/or resource versions:

```
==> helm upgrade --install stress-infra -n stress-infra --set stress-test-addons.env=prod --set deploy.chaosmesh=true /home/ben/sdk/azure-sdk-tools/tools/stress-cluster/cluster/kubernetes/stress-infrastructure                                                                                                                                                                                                                 
Error: UPGRADE FAILED: [unable to recognize "": no matches for kind "MutatingWebhookConfiguration" in version 
"admissionregistration.k8s.io/v1beta1", unable to recognize "": no matches for kind "ValidatingWebhookConfiguration" in version 
"admissionregistration.k8s.io/v1beta1"]                                                                                                                                             
Run: /home/ben/sdk/azure-sdk-tools/tools/stress-cluster/cluster/provision.ps1:21                                                                                                                                 
Line |                                                                                                                                                                                                             
21    |      Run @args                                                                                                                                                                                                 
        |      ~~~~~~~~~                                                                                                                                                                                                 
        | Command 'helm upgrade --install stress-infra -n stress-infra --set stress-test-addons.env=prod --set deploy.chaosmesh=true                                                                                     
        | /home/ben/sdk/azure-sdk-tools/tools/stress-cluster/cluster/kubernetes/stress-infrastructure' failed with code: 1               
```

### PR DESCRIPTION
**Pull request by ckairen**
#closes #2662

### CODE REVIEW

File: tools/stress-cluster/chaos/examples/network-stress-example/Chart.lock [hunk 1]
```diff
@@ -1,6 +1,6 @@
 dependencies:
 - name: stress-test-addons
   repository: https://stresstestcharts.blob.core.windows.net/helm/
-  version: 0.1.12
-digest: sha256:a2afbc89375d1518cd44194ca861e8a7a1ca1c16cfaea3409cdb48bfc910d878
-generated: "2021-11-16T19:17:24.5412749-05:00"
+  version: 0.1.14
```
Comment (benbp) [inline tools/stress-cluster/chaos/examples/network-stress-example/Chart.lock:4]: There are a couple Chart.lock files out of date compared to the Chart.yaml

File: tools/stress-cluster/cluster/kubernetes/stress-infrastructure/templates/stresswatcher.yaml [hunk 2]
```diff
@@ -1,36 +1,36 @@
...
-        imagePullPolicy: Always
-        image: azsdkengsys.azurecr.io/stress/watcher:{{ .Values.tag }}
-        command: ["dotnet", "Stress.Watcher.dll"]
-        {{- include "stress-test-addons.container-env" $ctx | nindent 8 }}
-      volumes:
-        # Volume template for mounting secrets
-        {{- include "stress-test-addons.env-volumes" $ctx | nindent 8 }}
-        # Volume template for mounting azure file share for debugging
-        {{- include "stress-test-addons.debug-file-volumes" $ctx | nindent 8 }}
+{{ $ctx := fromYaml (include "stress-test-addons.util.mergeStressContext" (list . "watcher" )) }}
```
Comment (benbp) [inline tools/stress-cluster/cluster/kubernetes/stress-infrastructure/templates/stresswatcher.yaml:1]: Given the full diff here, I'm suspecting either bad line endings were added, or they were finally removed?

File: tools/stress-cluster/cluster/provision.ps1 [hunk 3]
```diff
@@ -1,11 +1,12 @@
 param (
-    [string]$Environment = 'test',
+    [string]$Environment = 'dev',
```
Comment (benbp) [inline tools/stress-cluster/cluster/provision.ps1:2]: I think the answer is no, but worth double checking if any changes are necessary to the readme.

**Review by benbp — APPROVED** [review]

File: tools/stress-cluster/cluster/provision.ps1 (hunk 3 above)
Comment (ckairen) [reply tools/stress-cluster/cluster/provision.ps1:2]: Yup the answer is no

File: tools/stress-cluster/cluster/kubernetes/stress-infrastructure/templates/stresswatcher.yaml (hunk 2 above)
Comment (ckairen) [reply tools/stress-cluster/cluster/kubernetes/stress-infrastructure/templates/stresswatcher.yaml]: The LF/CRLF settings on my VSCode is a bit weird today, thanks for catching this

**Pull request merged**